### ISSUE TITLE
Error adding company - name is already taken.

### ISSUE THREAD

**Issue by oguzhankuzu** [grokability/snipe-it#10138]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hello there

Company, supplier, etc. in the SnipeIT interface. I am getting an error while adding. Actually I can add companies or suppliers but the information I added is not visible.

### Reproduction steps

1.snipeit
2.
3.
...

### Expected behavior

Hello there

Company, supplier, etc. in the SnipeIT interface. I am getting an error while adding. Actually I can add companies or suppliers but the information I added is not visible.

### Screenshots

Hello there

Company, supplier, etc. in the SnipeIT interface. I am getting an error while adding. Actually I can add companies or suppliers but the information I added is not visible.

### Snipe-IT Version

 Version v5.0.10 - build 5680 (master

### Operating System

Server 2019

### Web Server

IIS

### PHP Version

7.2.26 

### Operating System

Windows

### Browser

IE

### Version

11

**Comment by snipe** [comment]
Company names must be unique. I'm not sure what the issue is here.

**Comment by deltasquare4** [comment]
@snipe I have the same issue on `5.3.7`. I set up snipe it on a domain without TLS at first, and added details like company and location. After I enabled the TLS on the domain, I can no longer see the company and location I added before. When I try to add it, it says the name already exists.

I believe it's a cookie issue? Somehow Snipe-IT seems to process secure requests differently and would not return the results.

I can see the results if I visit the site through `http` protocol.
